Cartfile.resolved

```diff
@@ -0,0 +1 @@
+github "pointfreeco/swift-snapshot-testing" "1.5.0"
```

Comment:
1. Agreed. To be honest I was developing this feature with the Down projected nested in a demo iOS project so development would be more fluid. I notice there are some issues with Carthage when I try to build the Down project on it's own. I'll need to look into it more. One thing to note is that the snapshot library is available on Swift Package Manager, though the latest release has an issue here (see https://github.com/pointfreeco/swift-snapshot-testing/issues/244). A workaround is to checkout the master branch instead of the latest release. I'm curious to see if using SPM will be better. I'll get back to you on this.
2. Yeah... I failed to realize that travis is testing on all platforms. The snapshot library supports all platforms so it should be possible. I'll try to get this fixed.

Reply:
@johnxnguyen One thought for the multiple platforms: does it make sense to create a separate test target that only runs the snapshot tests on a specific platform for now and keep them apart from the unit tests?  This would simplify travis CI as well

Reply:
This is a great idea! I'm struggling to get the test target to compile / successfully run across all three platforms.